README.md

```diff
@@ -15,7 +15,11 @@ let license = require('rollup-plugin-node-license');
 rollup({
     ...
     plugins: [
-        license()
+        license({
+            // Format of the injected license comment(s).
+            // Options: "jsdocs" (default), or "table".
+            format: 'jsdocs'
```

Comment:
I'd change this from "jsdocs" to "jsdoc", to make it consistent with JSDoc.

Reply:
Ok, updated.